platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/ConfigurationAdminImpl.java

```diff
@@ -630,6 +630,9 @@ public ObjectClassDefinition getObjectClassDefinition(String pid) {
         if (haveOcd) {
           Service service = new ServiceImpl();
           service.setId(pid);
+          if(StringUtils.isNotEmpty(ocd.getDescription())) {
```

Comment:
✏️ Avoid retrieving the description twice. Call `ocd.getDescription()` once and cache it